diff --git a/metadatastore/conf.py b/metadatastore/conf.py
index 5637dc7..10badef 100644
--- filestore/conf.py
+++ filestore/conf.py
@@ -41,7 +41,9 @@ def load_configuration(name, prefix, fields):
     conf : dict
         Dictionary keyed on ``fields`` with the values extracted
     """
+    conda_path = '__CONDA_BUILD_PLACEHOLDER__'
     filenames = [
+        os.path.join(conda_path, name + '.yml'),
         os.path.join('/etc', name + '.yml'),
         os.path.join(os.path.expanduser('~'), '.config', name,
                      'connection.yml'),
