diff --git a/setup.py b/setup.py
index 2484be0..e553258 100644
--- setup.py
+++ setup.py
@@ -4,15 +4,6 @@ from __future__ import (absolute_import, division, print_function)
 import sys
 import warnings

-
-try:
-    from setuptools import setup
-except ImportError:
-    try:
-        from setuptools.core import setup
-    except ImportError:
-        from distutils.core import setup
-
 from distutils.core import setup

 MAJOR = 0
